.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ version: 2
 jobs:
   build_2.5_rails_5:
     docker:
-      - image: circleci/ruby:2.4.1
+      - image: circleci/ruby:2.7.4
```

Comment:
Worth adding a Ruby 3 build step?

Reply:
non-lts Rails 4 version and ruby 3 has compatibility issue resulting in issue with bundler version. Hence the build step fails to install bundler.